manifests/kiali-community/1.45.0/manifests/kiali.v1.45.0.clusterserviceversion.yaml

```diff
@@ -527,16 +527,6 @@ spec:
           - routes
           verbs:
           - get
-        - apiGroups: ["iter8.tools"]
-          resources:
-          - experiments
-          verbs:
-          - get
-          - list
-          - watch
-          - create
-          - delete
-          - patch
```

Comment:
I just noticed something. We already released 1.45 - see https://quay.io/repository/kiali/kiali?tab=tags and https://quay.io/repository/kiali/kiali-operator?tab=tags

That means these 1.45 CSV metadata files should be fixed to that release (and we need to publish these on operatorhub.io, which we have not done yet - we are only at 1.44 - see https://operatorhub.io/operator/kiali)

Therefore, we need to revert these changes to `manifests/kiali-community/1.45.0/manifests/kiali.v1.45.0.clusterserviceversion.yaml` and `manifests/kiali-upstream/1.45.0/manifests/kiali.v1.45.0.clusterserviceversion.yaml`

I'll create a new 1.46.0 version of both those files and then you can update them for this PR. I'll also have to publish 1.45 to operatorhub and the redhat registry.

Reply:
Yes, thanks for catching it.
This PR was created for 1.45 but now we need to move it.

Reply:
I updated my review comment above to explain what we need to do. I guess I should have just added a comment here. I'll do that now:

1. remove these changes to 1.45 OLM metadata
2. merge this PR with the new 1.46 OLM metadata - #478
3. Rebase your PR so you get the new 1.46 OLM metadata
4. Make your changes to the new 1.46 OLM metadata (i.e. the changes you revert from the 1.45 metadata need to be done to the 1.46 metadata).